plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/ext/provider/DockerExtConfBindingProvider.java

```diff
@@ -55,18 +61,22 @@ public String get() {
             return null;
         }
 
+        if (volumeOptions == null) {
+            volumeOptions="ro,Z";
+        }
+
         if (SystemInfo.isWindows()) {
             try {
                 final Path cheHome = WindowsHostUtils.ensureCheHomeExist();
                 final Path plgConfDir = cheHome.resolve(PLUGIN_CONF);
                 IoUtil.copy(extConfDir, plgConfDir.toFile(), null, true);
-                return plgConfDir.toString() + CONTAINER_TARGET;
+                return plgConfDir.toString() + CONTAINER_TARGET + ":" + volumeOptions;
             } catch (IOException e) {
                 LOG.warn(e.getMessage());
                 throw new RuntimeException(e);
             }
         } else {
-            return extConfDir.getAbsolutePath() + CONTAINER_TARGET;
+            return extConfDir.getAbsolutePath() + CONTAINER_TARGET + ":" + volumeOptions;
```

Comment:
small concern : maybe to have a getTargetOptions() helper method as there are some duplicates between unix and windows